plugins/modules/aci_switch_access_config.py

```diff
@@ -0,0 +1,216 @@
+                       "dn": "uni/tn-production",
+                       "name": "production",
+                       "nameAlias": "",
+                       "ownerKey": "",
+                       "ownerTag": ""
+                   }
+               }
+           }
+       ]
+   error:
+     description: The error information as returned from the APIC
+     returned: failure
+     type: dict
+     sample:
+       {
+           "code": "122",
+           "text": "unknown managed object class foo"
+       }
+   raw:
+     description: The raw output returned by the APIC REST API (xml or json)
+     returned: parse error
+     type: str
+     sample: '<?xml version="1.0" encoding="UTF-8"?><imdata totalCount="1"><error code="122" text="unknown managed object class "/></imdata>'
+   sent:
+     description: The actual/minimal configuration pushed to the APIC
+     returned: info
+     type: list
+     sample:
+       {
+           "fvTenant": {
+               "attributes": {
+                   "descr": "Production environment"
+               }
+           }
+       }
+   previous:
+     description: The original configuration from the APIC before the module has started
+     returned: info
+     type: list
+     sample:
+       [
+           {
+               "fvTenant": {
+                   "attributes": {
+                       "descr": "Production",
+                       "dn": "uni/tn-production",
+                       "name": "production",
+                       "nameAlias": "",
+                       "ownerKey": "",
+                       "ownerTag": ""
+                   }
+               }
+           }
+       ]
+   proposed:
+     description: The assembled configuration from the user-provided parameters
+     returned: info
+     type: dict
+     sample:
+       {
+           "fvTenant": {
+               "attributes": {
+                   "descr": "Production environment",
+                   "name": "production"
+               }
+           }
+       }
+   filter_string:
+     description: The filter string used for the request
+     returned: failure or debug
+     type: str
+     sample: ?rsp-prop-include=config-only
+   method:
+     description: The HTTP method used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: POST
+   response:
+     description: The HTTP response from the APIC
+     returned: failure or debug
+     type: str
+     sample: OK (30 bytes)
+   status:
+     description: The HTTP status from the APIC
+     returned: failure or debug
+     type: int
+     sample: 200
+   url:
+     description: The HTTP url used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: https://10.11.12.13/api/mo/uni/tn-production.json
+   """
+
+from ansible_collections.cisco.aci.plugins.module_utils.switch_config import SwitchConfig
+
+
+def main():
+    SwitchConfig("infraNodeConfig").main()
```

Comment:
I'm not familiar with these switch MOs on ACI but won't we run into an issue when we new attributes are introduced in switch access config that are not valid for switch fabric config?
Manipulating their respective child classes would be a an issue too.